### PR TITLE
[App Service] `az logicapp start/restart/stop`: Add new commands for logicapp

### PR DESCRIPTION
**Pull request by parth21999**
**Description**

Add `az logicapp start/restart/stop`  commands.

**Testing Guide**
First create a logicapp.
`az logicapp start` to start the logic app.
`az logicapp restart` to restart the logic app.
`az logicapp stop`  to stop the logic app.

---

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [X] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
App Service

File: src/azure-cli/azure/cli/command_modules/appservice/logicapp/_help.py [hunk 1]
```diff
@@ -49,3 +49,30 @@
     text: >
         az logicapp list --query "[?state=='Running']"
 """
+
+helps['logicapp restart'] = """
+type: command
+short-summary: Restart a logic app.
+examples:
+  - name: Restart a logic app. (autogenerated)
```
Comment (panchagnula) [inline src/azure-cli/azure/cli/command_modules/appservice/logicapp/_help.py:57]: NIT: Remove AutoGenerated text in the help for all 3 commands.

**Review by panchagnula — COMMENTED** [review]
In the files I see the Yaml files updated - but i don't see the test case updated on this PR?

**Comment by parth21999** [comment]
I'll remove the yaml file updates.

**Comment by panchagnula** [comment]
> I'll remove the yaml file updates.

Hmm Any reason we are not adding tests for these?

**Comment by parth21999** [comment]
> > I'll remove the yaml file updates.
> 
> Hmm Any reason we are not adding tests for these?

Logicapp is using the same `[start/stop/restart]_webapp` methods as webapp which have already have extensive tests. I can add some tests specifically for `az logicapp [start/stop/restart] if you like.

**Comment by panchagnula** [comment]
> > > I'll remove the yaml file updates.
> > 
> > 
> > Hmm Any reason we are not adding tests for these?
> 
> Logicapp is using the same `[start/stop/restart]_webapp` methods as webapp which have already have extensive tests. I can add some tests specifically for `az logicapp [start/stop/restart]` if you like.

yes we should add tests specifically for LogicApps - Thanks!

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**